src/helpers/apiHelpers.js

```diff
@@ -108,29 +107,9 @@ export const mapToId = (content) => {
 
 export const mapAPIContentToId = apiRes => mapToId(apiRes.content);
 
-const isWalletTransaction = actionType =>
-  actionType === 'transfer' ||
-  actionType === 'transfer_to_vesting' ||
-  actionType === 'cancel_transfer_from_savings' ||
-  actionType === 'transfer_from_savings' ||
-  actionType === 'transfer_to_savings' ||
-  actionType === 'delegate_vesting_shares' ||
-  actionType === 'claim_reward_balance';
+export const defaultAccountLimit = 2500;
```

Comment:
Isn't it too big? When it does fetch 2500 transactions my computer is having problems rendering them and scrolling is lagging. I would say ~100 or even smaller would be better for this.

Reply:
So since we are fetching all account history, it would be good for it to be a large number since sometimes, we don't get all the wallet transactions enough to fill the page  :(.. When I did the wallet page I played around with the numbers 1000, 2500, 5000, and even 10,000. But after discussing with @bonustrack we settled on 2500.